### PR TITLE
Updated vertical tab

### PR DESCRIPTION
**Pull request by MithunAdavelly**

### CODE REVIEW

**Comment by vanitaj** [comment]
Screen Shot 2019-10-07 at 3 51 44 PM
The active style is not matching with the zeplin styles.

**Comment by vanitaj** [comment]
Screen Shot 2019-10-07 at 3 55 00 PM
Active Style for horizantal tab is not matching with the zeplin styles

**Comment by vanitaj** [comment]
Changes required for vertical tabs - 
1) The hover should not have rounded corners on the right side.
2) Spacing between icon and label.

File: example/src/components/tabs/DarkModeTabs.jsx [hunk 1]
```diff
@@ -27,10 +27,38 @@ const styles = theme => ({
   root: {
     flexGrow: 1,
     backgroundColor: '#f5f5f5',
-    padding: '10px'
+    padding: '10px',
+    fontFamily: 'OpenSans',
+    fontStyle: 'normal',
+    fontStretch: 'normal',
+    LetterSpacing: 'normal'
```
Comment (vanitaj) [inline example/src/components/tabs/DarkModeTabs.jsx]: Please use camelCase for LetterSpacing

**Comment by vanitaj** [comment]
Changes required for vertical tabs -

The hover should not have rounded corners on the right side.
Spacing between icon and label.

File: example/src/components/tabs/DarkModeTabs.jsx (hunk 1 above)
Comment (MithunAdavelly) [reply example/src/components/tabs/DarkModeTabs.jsx]: Hi @vanitaj, fixed this and updated the PR over here with the latest commit. Thank you!

**Comment by MithunAdavelly** [comment]
> Changes required for vertical tabs -
> 
> 1. The hover should not have rounded corners on the right side.
> 2. Spacing between icon and label.

Hi @vanitaj, fixed this and updated the PR over here with the latest commit. Thank you!

File: src/components/CustomVerticalTabs.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+const VerticalTabs = withStyles(theme => ({
+  flexContainer: {
+    flexDirection: 'column'
+  },
+  indicator: {
+    display: 'none'
+  }
+}))(Tabs)
+
+const CustomVerticalTabs = ({ classes, children, ...other }) => (
```
Comment (vanitaj) [inline src/components/CustomVerticalTabs.jsx]: Can you rename it to VerticalTabs.

Comment (MithunAdavelly) [reply src/components/CustomVerticalTabs.jsx]: Hi @vanitaj, Updated the PR over here with the fix for latest review comments including the name change of this file and the hover styling of the icons and their following labels. Thank you!

**Pull request merged**